keel-core/src/main/kotlin/com/netflix/spinnaker/keel/lifecycle/LifecycleMonitor.kt

```diff
@@ -40,6 +23,10 @@ abstract class LifecycleMonitor(
 
   private val log by lazy { LoggerFactory.getLogger(javaClass) }
 
+  init {
+    log.debug("Lifecycle monitor ${javaClass.simpleName} enabled")
+  }
```

Comment:
Are you sure this will only get logged if it's enabled? (Assuming that's what you're trying to show)

Reply:
Well, there's no way to disable it, so .... yes? :D I kind of just want a confirmation that it's being used. I like those comforting log messages.